taxi_service/settings.py

```diff
@@ -79,6 +82,7 @@
     "default": {
         "ENGINE": "django.db.backends.sqlite3",
         "NAME": BASE_DIR / "db.sqlite3",
+        "DIRS": [BASE_DIR / "templates"],
```

Comment:
Code quality issue: It seems you have mistakenly added a DIRS key in the DATABASES configuration. This key is not recognized by Django and should be removed.